store/index.php

```diff
@@ -61,11 +61,20 @@
 <section class="grid">
     <div class="two-thirds">
         <h2>Support Development. Get Swag. Win Win.</h2>
-        <p>Every purchase goes towards developing elementary OS, its apps, and its services. We're a small <a href="/team">team</a>, mostly volunteer, working constantly to make elementary better. Every little bit of help is one step closer to hiring another full-time developer.</p>
-        <p>Due to COVID-19 some orders may experience delays. Please visit Printful's <a href="https://www.printful.com/covid-19" target="_blank" rel="noopener" title="Covid-19 updates for Printful customers">fullfillment estimate page</a> for more details.</p>
+        <p>Every purchase goes towards developing elementary OS, its apps, and its services. We're a small <a href="/team">team</a>, mostly volunteer, working constantly to make elementary better—your support helps make elementary OS more sustainable.</p>
```

Comment:
Removed the COVID paragraph here since we moved to an alert. At the same time, iterate on this copy just a tiny bit.